build.sh

```diff
@@ -1,9 +1,91 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
+set -e	# Stop on error
+cd "$(dirname "$0")"
 
-export VALVE_NO_AUTO_P4=1
-if [[ $1 == -v ]]; then
-  # Debug mode
-  make -f games.mak NO_CHROOT=1 MAKE_JOBS=1 MAKE_VERBOSE=1 "${@:2}"
+if pwd | grep -q " "; then
+	echo "You have cloned the source directory into a path with spaces"
+	echo "This will break a lot of thirdparty build scripts"
+	echo "Please move the source directory somewhere without a space in the path"
+	exit 1
+fi
+
+: "${CORES:=$(nproc)}"
+
+if [ ! -f ./thirdparty/gperftools-2.0/.libs/libtcmalloc_minimal.so ]; then
+	cd ./thirdparty/gperftools-2.0
```

Comment:
```suggestion
	cd thirdparty/gperftools-2.0
```

Reply:
I'll add this and other cleanups next weekend/when I have time.